.travis.yml

```diff
@@ -91,6 +91,8 @@ jobs:
             <<: *STANDARD_TEST_JOB
             stage: Test
             php: 7.2
+            before_script:
+                - php php-cs-fixer fix --rules @PHP71Migration,@PHP71Migration:risky,blank_line_after_opening_tag,native_function_invocation -q || travis_terminate 1
```

Comment:
i believe we can drop it from `php: nightly` now, what do you think? @kubawerlos @SpacePossum